src/log.cpp

```diff
@@ -136,7 +147,7 @@ void proj_log_debug (PJ *P, const char *fmt, ...) {
 ******************************************************************************/
     va_list args;
     va_start( args, fmt );
-    pj_vlog (pj_get_ctx (P), PJ_LOG_DEBUG_MAJOR , fmt, args);
+    pj_vlog (pj_get_ctx (P), PJ_LOG_DEBUG_MAJOR , add_short_name_prefix(P, fmt).c_str(), args);
```

Comment:
Just an off-hand comment now that we're here anyways. `PJ_LOG_DEBUG_MAJOR` and `PJ_LOG_DEBUG_MINOR` are specific to `proj_api.h` and could be removed altogether. . `PJ_LOG_DEBUG_MAJOR` -> `PJ_LOG_DEBUG` and . `PJ_LOG_DEBUG_MINOR` -> `PJ_LOG_TRACE` I believe.

Reply:
done